docs/javascript-api/index.md

```diff
@@ -65,10 +64,6 @@ async function build() {
 		// do some error reporting
 		console.error(error);
 	}
-	if (bundle) {
-		// closes the bundle
-		await bundle.close();
-	}
```

Comment:
While I agree that this should be in the docs, to my understanding this currently will only work if you are using TypeScript as NodeJS does not support the feature natively. Hence for now I fear we should rather present it as an alternative rather than the default until this is more widely supported. Otherwise I fear that users will complain, or overlook that fact that without explicit resource management, they need to manually close the bundle.

Reply:
I got it